lib/sqewer.rb

```diff
@@ -11,23 +11,9 @@ class Error < StandardError
     end
   end
 
-  # Sets an instance of Aws::SQS::Client to be used as a singleton.
-  # We recommend setting the options instance_profile_credentials_timeout and
-  # instance_profile_credentials_retries, for example:
-  #
-  #   sqs_client = Aws::SQS::Client.new(
-  #     instance_profile_credentials_timeout: 1,
-  #     instance_profile_credentials_retries: 5,
-  #   )
-  #   Storm.client = sqs_client
-  #
-  # @param client[Aws::SQS::Client] an instance of Aws::SQS::Client
-  def self.client=(client)
```

Comment:
I think it would still be useful to allow setting a custom client, what do you think?

Reply:
This is something that we added in the previous version to be able to test the singleton, so I think we can remove it now. I think nobody needed it before 🤔